spec/acceptance/alternative_port_spec.rb

```diff
@@ -2,27 +2,23 @@
 
 # These tests ensure that postgres can change itself to an alternative port
 # properly.
-describe 'postgresql::server', :unless => UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+describe 'postgresql::server', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
   it 'on an alternative port' do
-    pp = <<-EOS
+    pp = <<-MANIFEST
       class { 'postgresql::server': port => '55433' }
-    EOS
+    MANIFEST
 
-    apply_manifest(pp, :catch_failures => true)
-    apply_manifest(pp, :catch_changes => true)
+    apply_manifest(pp, catch_failures: true)
+    apply_manifest(pp, catch_changes: true)
   end
 
-  describe port(55433) do
+  describe port(55_433) do
```

Comment:
Is this port number correct?

Reply:
Rubocop states that Numeric Literals should be separated into lengths of no more than 3 digit's each, with underscores in the place of decimals.

Reply:
I would suggest changing it back and disabling the rule for this line.